package.json

```diff
@@ -81,12 +81,12 @@
   },
   "dependencies": {
     "classnames": "^2.2.0",
-    "cozy-ui": "^3.0.0-beta24",
+    "cozy-ui": "beta32",
     "date-fns": "^1.27.2",
     "node-polyglot": "^2.2.0",
     "piwik-react-router": "^0.8.2",
-    "preact": "^7.1.0",
-    "preact-compat": "^3.9.0",
+    "preact": "^8.1.0",
```

Comment:
Preact 8.X introduced a different way of handling scroll events, and that made react-virtualized fail at handling long lists in Drive. That's the reason I reverted Drive's preact version to 7.X